diff --git a/solr-client/src/main/java/org/kuali/ole/controller/MainController.java b/solr-client/src/main/java/org/kuali/ole/controller/MainController.java
index 8e756c8..3bcd4bf 100644
--- a/solr-client/src/main/java/org/kuali/ole/controller/MainController.java
+++ b/solr-client/src/main/java/org/kuali/ole/controller/MainController.java
@@ -21,6 +21,7 @@ import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.format.annotation.DateTimeFormat;
 import org.springframework.scheduling.annotation.EnableAsync;
 import org.springframework.ui.Model;
 import org.springframework.util.StopWatch;
@@ -115,7 +116,7 @@ public class MainController {
 
     @ResponseBody
     @RequestMapping(value="/partialIndexByDate", method = RequestMethod.POST, produces = "application/json")
-    public PartialIndexStatus partialIndexByDate(@RequestParam("fromDate") Date date,@RequestParam("docPerThread") Integer docPerThread, @RequestParam("numberOfThreads") Integer numberOfThreads, HttpServletRequest request) {
+        public PartialIndexStatus partialIndexByDate(@RequestParam("fromDate") @DateTimeFormat(pattern="yyyy-MM-dd") Date date,@RequestParam("docPerThread") Integer docPerThread, @RequestParam("numberOfThreads") Integer numberOfThreads, HttpServletRequest request) {
         return partialIndexService.getPartialIndexStatus(date, docPerThread, numberOfThreads);
 
     }
diff --git a/solr-client/src/main/java/org/kuali/ole/security/WebSecurityConfig.java b/solr-client/src/main/java/org/kuali/ole/security/WebSecurityConfig.java
index d31bdbd..306f4f6 100644
--- a/solr-client/src/main/java/org/kuali/ole/security/WebSecurityConfig.java
+++ b/solr-client/src/main/java/org/kuali/ole/security/WebSecurityConfig.java
@@ -29,9 +29,7 @@ public class WebSecurityConfig extends WebSecurityConfigurerAdapter {
                 .loginPage("/index")
                 .permitAll()
                 .and()
-                .csrf()
-                .csrfTokenRepository(CookieCsrfTokenRepository.withHttpOnlyFalse());
-
+                .csrf().disable();
     }
 
     @Override
diff --git a/solr-client/src/main/resources/static/js/partialIndex.js b/solr-client/src/main/resources/static/js/partialIndex.js
index e361e15..9311835 100644
--- a/solr-client/src/main/resources/static/js/partialIndex.js
+++ b/solr-client/src/main/resources/static/js/partialIndex.js
@@ -68,7 +68,7 @@ angular.module("oleSolrClient.partialIndex", ['ui.bootstrap', 'ngStorage', 'ui.b
                     });
                 }else if ($scope.partialIndexRequest.type === 'indexByDate') {
 
-                    var fromDate = $scope.partialIndexRequest.fromDate;
+                    var fromDate = $filter('date')($scope.partialIndexRequest.fromDate,'yyyy-MM-dd');
                     var numberOfThreads = $scope.partialIndexRequest.noOfDbThreads;
                     var docPerThread = $scope.partialIndexRequest.docsPerThread;
                     fd.append('fromDate', fromDate);
